src/services/api.js

```diff
@@ -1,7 +1,7 @@
-// const API_ROOT = `http://localhost:3000/api/v1`;
+const API_ROOT = `http://localhost:3000/api/v1`;
 const CORS_PROXY = `https://floating-wildwood-76961.herokuapp.com/`;
 const AFFIRMATIONS_API = `https://www.affirmations.dev`;
-const API_ROOT = `https://carecast-app-api.herokuapp.com/api/v1`;
+// const API_ROOT = `https://carecast-app-api.herokuapp.com/api/v1`;
```

Comment:
I suspect you don't want to commit this. I recommend putting this in your `.env` folder at some point, which helps so you don't have to constantly toggle between the two